index.html

```diff
@@ -48,5 +48,116 @@ <h2>I am a Software Developer</h2>
       </div>
     </section>
 
+    <section class="work-section">
+      <div class="title">
+  <h1>My Recent Works</h1>
+      </div>
+    </section>
+
+ <section class="works">
+   <div class="my-works">
+     <div class="work-pic-1">
+       <img src="./img/work-img/work-1.png" alt="My work" />
+     </div>
+     <div class="work-content">
+       <h2>Multi-Post Stories</h2>
+       <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+       <div class="programming-languages">
+         <p>html</p>
+         <p>Bootstrap</p>
+         <p>Ruby</p>
+       </div>
+       <a href="#">See Project</a>
+     </div>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>
+   </section>
+
+   <section class="About-me">
+     <h2>About me</h2>
+     <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <a href="#">Get My Resume</a>

```

Comment:
- [ ] I suggest you follow the Figma as close as possible for this part of the project aswell.

Your design 
![Screenshot from 2022-05-28 16-56-21](https://user-images.githubusercontent.com/52143815/170830894-8287e841-f184-4e22-bb8b-b4eb322d416e.png)

The figma design
![Screenshot from 2022-05-28 16-59-34](https://user-images.githubusercontent.com/52143815/170830936-a7d2f031-714b-4caf-adfa-fb5727df8901.png)